src/utils.ts

```diff
@@ -414,11 +414,11 @@ export async function generateTemplateFile(
  * and open the IoT Workspace Project. Otherwise ask and New IoT Project.
  */
 export async function handleIoTWorkspaceProjectFolder(
-    telemetryContext: TelemetryContext): Promise<boolean> {
+    telemetryContext: TelemetryContext) {
   if (!(vscode.workspace.workspaceFolders &&
         vscode.workspace.workspaceFolders.length > 0) ||
       !vscode.workspace.workspaceFolders[0]) {
-    return false;
+    return;
```

Comment:
Since the function returns from boolean to void, is there any code depending on the return value?

Reply:
No caller function check the return value. That's why I modify the return value.